src/macos.rs

```diff
@@ -106,7 +106,7 @@ pub mod system_fonts {
     }
 
     /// Query the names of all fonts installed in the system
-    pub fn query_all() -> Vec<String> {
+    pub fn query_all() -> Result<Vec<String>, Error> {
```

Comment:
Import / Definition for Error missing, see [travis](https://travis-ci.org/MSleepyPanda/rust-font-loader/jobs/331545202#L135)